internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
```

Comment:
No error handling is present when defining the routes. While setting up routes typically does not error, it’s good practice to ensure handlers are properly registered and manageable. Consider wrapping your route registration or controller initialization in a way that can handle potential errors, or at the very least log errors if they occur during initialization.